packages/suite/src/components/suite/labeling/WalletLabeling.tsx

```diff
@@ -18,10 +18,12 @@ export const useWalletLabeling = () => {
     const { translationString } = useTranslation();
 
     const defaultAccountLabelString = useCallback(
-        ({ device }: { device: TrezorDevice }) =>
-            device.useEmptyPassphrase
-                ? translationString('TR_NO_PASSPHRASE_WALLET')
-                : translationString('TR_PASSPHRASE_WALLET', { id: device.walletNumber }),
+        ({ device }: { device: TrezorDevice }) => {
+            if (device.useEmptyPassphrase) return translationString('TR_NO_PASSPHRASE_WALLET');
+            if (!device.walletNumber) return undefined;
```

Comment:
Missing this leads to names like "Hidden wallet #"

Reply:
nit: I really prefer `{}` everywhere